websocketOutportDriver/sender/acknowledgesHolder.go

```diff
@@ -0,0 +1,45 @@
+package sender
+
+import "sync"
+
+type acknowledgesHolder struct {
+	acknowledges map[string]*websocketClientAcknowledgesHolder
+	mut          sync.Mutex
+}
+
+// NewAcknowledgesHolder returns a new instance of acknowledgesHolder
+func NewAcknowledgesHolder() *acknowledgesHolder {
```

Comment:
:+1:right. We have small, one-purpose structs that properly manage the concurrent operations by themselves.
They only expose the needed operations and the used of this struct doesn't care it manipulates maps as to take care not running into concurrent accesses.